notebooks/demo1-create-tables.ipynb

```diff
@@ -0,0 +1,516 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9b0596bc",
+   "metadata": {},
+   "source": [
+    "# Data Ingestion\n",
+    "\n",
+    "This notebook is the first in a series of notebooks created for the Nimbus demo. Here, we will show how to read raw data files from an s3 storage, and ingest them as a table into Trino. Such a table can then be used for further analysis or for creating visualizations in Apache Superset.\n",
+    "\n",
+    "The notebook also shows how to join two tables on Trino to create a new table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "244e7aec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import re\n",
+    "import pathlib\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "import boto3\n",
+    "import trino\n",
+    "\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "384e1ace",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Injecting Credentials\n",
+    "\n",
+    "To inject the required access credentials (for S3 and Trino) into this notebook, we will define them as environment variables in a `credentials.env` file, and load them using dotenv. An example of what the contents of `credentials.env` could look like is shown below\n",
```

Comment:
This is true if you are exploring the notebook interactively. But shouldn't there be a different flow if the notebook is used in automation where the variables are defined by an openshift secret and not a .env file?

Reply:
It does not HAVE to be in this PR, in can be in the next. But this notebook nor the PR note gives any indication that you plan on including the automation flow, which is an important part of this project overall. 

It might be sufficient to just update the markdown to make clear that this will still work in automation with opc secrets that will define the env variables, right?

Reply:
+1 updated the markdown and the PR note.